test/unit/test_sdb.c

```diff
@@ -154,6 +154,43 @@ bool test_sdb_namespace(void) {
 	mu_end;
 }
 
+static bool foreach_filter_user_cb(void *user, const char *key, const char *val) {
+	Sdb *db = (Sdb *)user;
+	const char *v = sdb_const_get(db, key, NULL);
```

Comment:
isn't this the same as `val` itself?

Reply:
It is, but we specifically access `db` here to test that the `void *user` pointer is passed succesfully.

Reply:
gotcha.